samples/drivers/dac/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  name: DAC driver sample
+tests:
+  sample.drivers.dac:
+    tags: DAC
+    depends_on: dac
+    harness: console
+    harness_config:
+      type: one_line
+      regex:
+        - "Counter received: (.*)"
```

Comment:
Does not macth sample output:
```
DAC internal reference voltage: 3300 mV                                         
Generating sawtooth signal at DAC channel 1.  
```
Btw, if using multi_line, also add `ordered: true`